draft-ietf-quic-ack-frequency.md

```diff
@@ -162,6 +162,10 @@ receiving ACK_FREQUENCY frames.  If an endpoint sends the transport parameter,
 the peer is allowed to send ACK_FREQUENCY frames independent of whether it also
 sends the min_ack_delay transport parameter or not.
 
+Endpoints MUST NOT remember the value of the min_ack_delay transport parameter
+they received. Due to this, ACK_FREQUENCY frames MUST NOT be sent in 0-RTT
```

Comment:
```suggestion
they received. Consequently, ACK_FREQUENCY frames MUST NOT be sent in 0-RTT
```

Reply:
isn't it "cannot"?

(You see, if we say "MUST NOT", we probably need to say what happens if you get a 0-RTT packet with ACK_FREQUENCY in it, but I know that some implementations can't really make that distinction.)

Reply:
Good point, I think cannot is more correct, updated.